client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRConfig.java

```diff
@@ -18,51 +18,90 @@
+      MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_SEND_CHECK_INTERVAL_MS;
+  public static final long RSS_CLIENT_SEND_CHECK_INTERVAL_MS_DEFAULT_VALUE =
+      RssClientConfig.RSS_CLIENT_SEND_CHECK_INTERVAL_MS_DEFAULT_VALUE;
+  public static final String RSS_CLIENT_SEND_CHECK_TIMEOUT_MS =
+      MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_SEND_CHECK_TIMEOUT_MS;
+  public static final long RSS_CLIENT_SEND_CHECK_TIMEOUT_MS_DEFAULT_VALUE =
+      RssClientConfig.RSS_CLIENT_SEND_CHECK_TIMEOUT_MS_DEFAULT_VALUE;
+  public static final String RSS_CLIENT_BITMAP_NUM = MR_RSS_CONFIG_PREFIX + "rss.client.bitmap.num";
   public static final int RSS_CLIENT_DEFAULT_BITMAP_NUM = 1;
-  public static final String RSS_CLIENT_MAX_SEGMENT_SIZE = "mapreduce.rss.client.max.buffer.size";
+  public static final String RSS_CLIENT_MAX_SEGMENT_SIZE =
+      MR_RSS_CONFIG_PREFIX + "rss.client.max.buffer.size";
   public static final long RSS_CLIENT_DEFAULT_MAX_SEGMENT_SIZE = 3 * 1024;
-  public static final String RSS_STORAGE_TYPE = "mapreduce.rss.storage.type";
+  public static final String RSS_STORAGE_TYPE = MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_STORAGE_TYPE;
+
+  public static final String RSS_PARTITION_NUM_PER_RANGE =
+      MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_PARTITION_NUM_PER_RANGE;
+  public static final int RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE =
+      RssClientConfig.RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE;
+  public static final String RSS_REMOTE_STORAGE_PATH =
+      MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_REMOTE_STORAGE_PATH;
+  public static final String RSS_INDEX_READ_LIMIT =
+      MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_INDEX_READ_LIMIT;
+  public static final int RSS_INDEX_READ_LIMIT_DEFAULT_VALUE =
+      RssClientConfig.RSS_INDEX_READ_LIMIT_DEFAULT_VALUE;
+  public static String RSS_CLIENT_READ_BUFFER_SIZE =
+      MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_READ_BUFFER_SIZE;
 
-  public static final String RSS_PARTITION_NUM_PER_RANGE = "mapreduce.rss.partitionNum.per.range";
-  public static final int RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE = 1;
-  public static final String RSS_BASE_PATH = "mapreduce.rss.base.path";
-  public static final String RSS_INDEX_READ_LIMIT = "mapreduce.rss.index.read.limit";
-  public static final int RSS_INDEX_READ_LIMIT_DEFAULT_VALUE = 500;
-  public static String RSS_CLIENT_READ_BUFFER_SIZE = "mapreduce.rss.client.read.buffer.size";
   // When the size of read buffer reaches the half of JVM region (i.e., 32m),
   // it will incur humongous allocation, so we set it to 14m.
-  public static String RSS_CLIENT_READ_BUFFER_SIZE_DEFAULT_VALUE = "14m";
+  public static String RSS_CLIENT_READ_BUFFER_SIZE_DEFAULT_VALUE =
+      RssClientConfig.RSS_CLIENT_READ_BUFFER_SIZE_DEFAULT_VALUE;
 
-  public static String RSS_DYNAMIC_CLIENT_CONF_ENABLED = "mapreduce.rss.dynamicClientConf.enabled";
+  public static String RSS_DYNAMIC_CLIENT_CONF_ENABLED =
```

Comment:
Do we need to put "rss.dynamicClientConf.enabled" in RssClientConfig?

Reply:
ok